dll/win32/shlwapi/utils.cpp

```diff
@@ -140,16 +140,34 @@ SHLWAPI_IsBogusHRESULT(HRESULT hr)
     return (hr == E_FAIL || hr == E_INVALIDARG || hr == E_NOTIMPL);
 }
 
+// Used for IShellFolder_GetDisplayNameOf
+struct RETRY_DATA
+{
+    UINT uRemove;
+    UINT uAdd;
+    DWORD dwRetryFlags;
+};
+static const RETRY_DATA g_RetryData[] =
+{
+    { SHGDN_FOREDITING,    0,                SFGDNO_RETRYALWAYS         },
+    { SHGDN_FORADDRESSBAR, 0,                SFGDNO_RETRYALWAYS         },
+    { 0,                   SHGDN_FORPARSING, SFGDNO_RETRYALWAYS         },
+    { SHGDN_FORPARSING,    0,                SFGDNO_RETRYWITHFORPARSING },
+    { SHGDN_INFOLDER,      0,                SFGDNO_RETRYALWAYS         },
+};
+
 /*************************************************************************
  * IShellFolder_GetDisplayNameOf [SHLWAPI.316]
+ *
+ * @note Don't confuse with <shobjidl.h> inline function of same name.
```

Comment:
```suggestion
 * @note Don't confuse with <shobjidl.h> inline function of the same name.
```
Maybe you could use this opportunity to tell in the note what both of these functions are for (I mean, what differ between the two).

Reply:
Fixed. The difference written.